draft-ietf-quic-qpack.md

```diff
@@ -587,9 +587,10 @@ entries might cause a stream to become blocked, as described in
 ~~~~~~~~~~
 {:#fig-size-sync title="Table State Synchronize"}
 
-An encoder that receives an Insert Count that is greater than the number of
-dynamic table entries beyond the current Largest Known Received entry MUST treat
-this as a connection error of type `HTTP_QPACK_DECODER_STREAM_ERROR`.
+An encoder that receives an Insert Count equal to zero, or that is greater than
```

Comment:
I don't like the "that" in "or that:" does it refer to the insert count or to the encoder?

Reply:
Perhaps something along the lines of:

> The Insert Count value cannot be zero and it cannot be greater than the current Largest Known Received value.  An encoder that receives an invalid Insert Count value MUST treat it...

Reply:
The constraint is tighter than < Largest Known Received, the text here captures it.  LKR + IC must be < Base Index

Reply:
OK, mine was just an example.  I'd split the sentence into two to make it unambiguous and easier to read.